game/scripts/vscripts/Sections/Chapter2/SectionCreeps.ts

```diff
@@ -116,12 +115,15 @@ const onStart = (complete: () => void) => {
                     tg.neverComplete()
                 ]),
                 tg.seq([
-                    tg.setCameraTarget(radiantCreeps[0]),
-                    tg.wait(3.5),
+                    tg.wait(2.5),
```

Comment:
Is this change necessary? 3.5 is more or less the time it takes to the creeps to meet and start fighting.

Reply:
The panCamera step that I added takes about a second or so, so with this adjustment the dialogue syncs well with the creeps connecting